spec.md

```diff
@@ -534,23 +534,50 @@ Credentials are a set of information used by an application or a user to utilize
 
 #### Log Drain ####
 
-There are a class of service offerings that provide aggregation, indexing, and analysis of log data. To utilize these services an application that generates logs needs information for the location to which it will stream logs. If a broker represents one of these services, it MAY return a `syslog_drain_url` in the response for a request to create a service binding, to which logs MUST be streamed.
+There are a class of service offerings that provide aggregation, indexing, and
+analysis of log data. To utilize these services an application that generates
+logs needs information for the location to which it will stream logs. A
+create binding response from one of these services MUST include a
+`syslog_drain_url`. This URL MUST be used when sending logs to the service.
```

Comment:
I think it should say

```
There are a class of service offerings that provide aggregation, indexing, and
analysis of log data. To utilize these services an application that generates 
logs needs information for the location to which it will stream logs. 
A create binding response from a service broker that provides one of these services 
MUST include a `syslog_drain_url`. The platform MUST use `syslog_drain_url` when
sending logs to the service.
```

My only question Is it the platform that streams the logs to this service or the application? @zrob may know..